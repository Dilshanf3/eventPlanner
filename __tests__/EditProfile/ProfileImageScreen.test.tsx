// ProfileImageScreen.test.tsx

import React from 'react';
import {render} from '@testing-library/react-native';
import {NavigationContainer} from '@react-navigation/native'; // Import NavigationContainer
import ProfileImageScreen from '../../src/screens/Auth/EditProfileImage/EditProfileImageScreen';

describe('<ProfileImageScreen />', () => {
  it('renders correctly', () => {
    const {toJSON} = render(
      <NavigationContainer>
        <ProfileImageScreen />
      </NavigationContainer>,
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
