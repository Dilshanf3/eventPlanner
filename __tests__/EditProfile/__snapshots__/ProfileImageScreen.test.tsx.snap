// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileImageScreen /> renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
      "paddingHorizontal": 20,
    }
  }
>
  <Text
    style={
      {
        "color": "#1A1A1A",
        "fontSize": 32,
        "fontWeight": "600",
        "marginBottom": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome
  </Text>
  <Text
    style={
      {
        "color": "#6D6D6D",
        "fontSize": 16,
        "marginBottom": 40,
        "textAlign": "center",
      }
    }
  >
    You are logged in for the first time and can upload a profile photo.
  </Text>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "alignSelf": "center",
        "backgroundColor": "#F4E6E2",
        "borderRadius": 75,
        "height": 150,
        "justifyContent": "center",
        "opacity": 1,
        "width": 150,
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <Image
        source={
          {
            "testUri": "../../../src/assets/images/Union.png",
          }
        }
        style={
          {
            "height": 40,
            "resizeMode": "contain",
            "tintColor": "#D36F56",
            "width": 40,
          }
        }
      />
    </View>
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
        "marginTop": "80%",
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": false,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#DA5E42",
          "borderRadius": 5,
          "flexDirection": "row",
          "justifyContent": "center",
          "marginTop": 20,
          "opacity": 1,
          "paddingVertical": 15,
          "width": "100%",
        }
      }
    >
      <Text
        style={
          [
            {
              "color": "#fff",
              "fontSize": 16,
              "marginRight": 10,
            },
            undefined,
          ]
        }
      >
        Continue
      </Text>
      <Image
        source={
          {
            "testUri": "../../../src/assets/images/arrow.png",
          }
        }
        style={
          {
            "height": 24,
            "resizeMode": "contain",
            "width": 24,
          }
        }
      />
    </View>
  </View>
</View>
`;
